controller/konnect/reconciler_generic.go

```diff
@@ -572,9 +573,10 @@ func setServerURLAndOrgIDFromAPIAuthConfiguration(
 	ent interface {
 		GetKonnectStatus() *konnectv1alpha1.KonnectEntityStatus
 	},
+	serverURL ops.ServerURL,
 	apiAuth konnectv1alpha1.KonnectAPIAuthConfiguration,
```

Comment:
Given that we're not extracting server URL from provided `apiAuth` anymore and we only need it to get the org ID I believe we can change the 3rd arg to be just that: orgID string. WDYT?